sway-core/src/type_system/type_info.rs

```diff
@@ -178,6 +185,9 @@ impl Hash for TypeInfo {
                 look_up_type_id(*elem_ty).hash(state);
                 count.hash(state);
             }
+            TypeInfo::RawUntypedPtr => {
+                state.write_u8(20);
```

Comment:
Should this just be `19`?

Reply:
Why not `18`?

Reply:
Good question. I think I forgot what comes after 17.